platform/src/test/java/org/exoplatform/platform/qa/ui/platform/wiki/WikiInformationTestIT.java

```diff
@@ -0,0 +1,298 @@
+
+    info("Create child 2 for wiki page");
+    wikiHomePage.selectAPage(title);
+    wikiHomePage.goToAddBlankPage();
+    richTextEditor.addSimplePage(child2, child2);
+    wikiManagement.saveAddPage();
+    $(byText(child2)).should(Condition.exist);
+
+    info("Open Page info");
+    String date = getDateByTextFormat("MM dd, yyyy");
+    wikiHomePage.goToPageInformation(title);
+
+    info("Summary: Title, author, Last changed by");
+    $(byText("Title")).parent().parent().find(byText(title)).should(Condition.exist);
+    $(byText("Author")).parent().parent().find(byText("Root Root")).should(Condition.exist);
+    $(byText("Last changed by")).parent().parent().find(byText("Root Root")).should(Condition.exist);
+
+    info("Related page: list all related page. A button [Add More Relation]");
+    $(byClassName("empty center")).find(byText("Empty data"));
+    $(ELEMENT_PAGE_INFO_ADD_MORE_RELATIONS).waitUntil(Condition.appears, Configuration.timeout);
+
+    info("Hierarchy: list parent page and child");
+    $(byClassName("uiPageInfoItem uiPageInfoHierarchy")).find(byClassName("nodeLabel")).find(byText("Wiki Home"));
+    $(byClassName("uiPageInfoItem uiPageInfoHierarchy")).find(byClassName("nodeGroup")).find(byText("child1"));
+    $(byClassName("uiPageInfoItem uiPageInfoHierarchy")).find(byClassName("nodeGroup")).find(byText("child2"));
+
+    info("Recent changes: list all revisions and a button [View Page History]");
+    $(byClassName("")).find(byText(""));
+    $(ELEMENT_VIEW_PAGE_HISTORY).should(Condition.exist);
+
+    info("Delete the page");
+    homePagePlatform.goToWiki();
+    wikiHomePage.deleteWiki(title);
+  }
+
+  @Test
+  @BugInPLF("WIKI-1313")
+
+  public void test02_AddRelationWithIntranetPortal() {
+    info("Test 02: Add relations with Intranet portal");
+    String space1 = "space1" + getRandomNumber();
+    String space2 = "space2" + getRandomNumber();
+    String title1 = "title1" + getRandomNumber();
+    String title2 = "title2" + getRandomNumber();
+
+    /*
+     * Step Number: 1 Step Name: Step 1: Add new space Step Description: - Login
+     * portal - Click join a space - Add new space for "Space 1" and "Space 2" Input
+     * Data: Expected Outcome: New space is added successfully
+     */
+
+    /*
+     * Step Number: 2 Step Name: Step 2: Add new page for wiki on space 1 and space
+     * 2 Step Description: - Go to Space 1/ wiki and Space2/wiki - Add new page for
+     * wiki with name "Page1" and "Page2" Input Data: Expected Outcome: Add new page
+     * successfully
+     */
+
+    info("Create space 1 and wiki page 1");
+    homePagePlatform.goToMySpaces();
+    spaceManagement.addNewSpaceSimple(space1, space1);
+    info("Add new wiki page for space 1");
+    spaceHomePage.goToWikiTab();
+    wikiHomePage.goToAddBlankPage();
+    richTextEditor.addSimplePage(title1, title1);
+    wikiManagement.saveAddPage();
+    $(byText(title1)).should(Condition.exist);
```

Comment:
In this case, the use of byText will not cause problems because it's variable and we must verify the page's creation by it's title